src/pylogctx/celery.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import absolute_import
+
+from celery.app import current_task
+from celery.app.task import Task
+
+from .core import context
+
+
+class LoggingTask(Task):
+    def __call__(self, *args, **kwargs):
+        task = current_task()
+        context.update(
+            celeryTask=task.name,
+            celeryTaskId=task.request.id,
```

Comment:
I too prefer the `snake_case` for readability. And also because of snakes ;-)

But consistency is good. That's hard to have `thread_id`, `threadId`, etc. in log server. Consistency is important.